src/main/java/com/example/java/maven/gameBlackJack/Game.java

```diff
@@ -0,0 +1,96 @@
+package com.example.java.maven.gameBlackJack;
+
+public class Game {
+    Player player = new Player(UserInputProvider.getPlayersName());
+    Croupier croupier = new Croupier();
+    private final Deck deck;
+    private final static String INSTRUCTIONS =
```

Comment:
you can use text block:
https://mkyong.com/java/java-multi-line-string-text-blocks/
7. Text Blocks (Preview in Java 13, 14)

Reply:
Done